dashboard/config/blocks/GamelabJr/gamelab_firstTouched.json

```diff
@@ -0,0 +1,15 @@
+{
+  "category": "Sprites",
+  "config": {
+    "color": [
+      312,
+      0.32,
+      0.62
+    ],
+    "expression": "arguments[0]",
+    "orderPrecedence": 1,
```

Comment:
Why do we need to override the default precedence?  Can we infer this from other block info?

Reply:
In this case the order precedence is dictated by the `[]` operator in `expression`. We could theoretically parse that field and figure it out from that, but that seems like overkill.

People haven't really been using expression blocks, so this hasn't been an issue. Worst case scenario is that they omit this field from an expression block and we include a few extra parens in the generated code.

Reply:
Cool, makes sense.